src/telegraf.ts

```diff
@@ -142,7 +200,19 @@ export class Telegraf<C extends Context = Context> extends Composer<C> {
     return this
   }
 
+  initializeQueue(concurrency: boolean | number) {
```

Comment:
```suggestion
  private initializeQueue(concurrency: boolean | number) {
```

Reply:
The problem here is that we need to initialize the queue for running the tests. Given that we know webhook/long polling only upon `bot.launch`, we cannot construct the object when constructing the bot. As a result, it needs to be constructed manually by the test suite (which does not `launch` the bot).

Any suggestions how to improve this? We could look into making the concurrency value adjustable during runtime, but I'm not sure what this requires in turn.

Reply:
Not a problem, tests aren't typechecked :) Don't worry about it now, it can always be made `public` if needed.